ui/docs/css.md

```diff
@@ -0,0 +1,39 @@
+# CSS/SCSS
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Guidelines](#guidelines)
+  - [Helper classes](#helper-classes)
+  - [Core class styles](#core-class-styles)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Guidelines
+
+- [**Helper classes**](#helper-classes) should be used if a styling block does not already exist and/or a reasonable number of helper classes can cover the desired style.
+- [**Core classes**](#core-class-styles) provide styling for any classes not associated with a component. The scope of each file is defined by the file name.
+- **Component specific styling** should only be added to, or created when you cannot achieve the styling with a helper class or core class.
+- **Utils'** files define mixins, and variables. 
+
+> ### Known issues
+> The following are known issues that we are working to address in conjunction with the adoption of HDS.
+> 1. **Size variables** The UI does not follow a consistent size variable pattern. We use both px and rem to define font-size and we use px, rem, and ems to define margins, heights and widths. For accessibility reasons we _should_ define all font-sizing at the very least by rem, though this is not consistently done in the app.
+> 2. **Variable naming** The UI does not have a consistent pattern to variable naming. We use a mix of numbers and words (ex: `ui-gray-050` is the same as `ui-gray-lightest`).
+> 3. **Random variables** We have dieing but not dead variables. For example, we have some variables that define box-shadows and we have some variables to define animations, but we are missing many box-shadow definitions and we do not consistently use the animation variables.
+> 4. **Missing variables** The UI does not have a variable for all commonly occurring sizes and colors. For example, we do not have a variable that covers the `14px` though it is a commonly used size.
+> 5. **!Important** `!important` is sprinkled throughout helper, core and component files. Ideally, the cascading and order of styles would eliminate the need of this key-word. However, because `!important` exists randomly in many of our files, we now have cascading issues inside helper files and core files. In all known cases where these issues exists a comment has been left as to why the order of classes in that particular area matters.
```

Comment:
Small nit on the grammar here, _exists_ should be singular:

```suggestion
> 5. **!Important** `!important` is sprinkled throughout helper, core and component files. Ideally, the cascading and order of styles would eliminate the need of this keyword. However, because `!important` exists randomly in many of our files, we now have cascading issues inside helper files and core files. In all known cases where these issues exist, a comment has been left as to why the order of classes in that particular area matters.
```